### PR TITLE
update command args

### PR DESCRIPTION
**Pull request by bartr**
update command args in prep for merging with LodeRunner

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,12 +209,26 @@ We use the `--json-log` command line option to integrate Docker container logs w
...
   - default location current directory
   - `required`
-- --webv-prefix string
-  - prefix to add to server values that don't begin with http
-  - default `https://`
-- --webv-suffix string
-  - suffix to add to server values that don't begin with http
-  - default `.azurewebsites.net`
+- -u --base-url
+  - base URL of test files using http
```
Comment (jofultz) [inline README.md:213]: Should we write something more like:
> base URL (protocol + domain + path) without any file specified where the all test files may be found

Also, the example shows https, but the text states http.  Should we specify one or write it as either?

File: src/README.md [hunk 2]
```diff
@@ -14,6 +14,15 @@ Web Validate (WebV) is a web request validation tool that we use to run end-to-e
...
+  - use `--log-format json` instead (starting with this release)
+
+- `--summary-minutes` is deprecated in v2.0
+  - use some type of log to store and summarize the results
+
+- `--max-concurrent` is deprecated in v2.0
+  - use `--sleep` and `--timeout` to control connections
+
 - Test files must migrate to the new json format
 - This is a breaking change in the 2.0 release
```
Comment (jofultz) [inline src/README.md:27]: Maybe bold to draw attention:

- **This is a breaking change in the 2.0 release**

File: src/app/Program.cs [hunk 3]
```diff
@@ -36,42 +37,52 @@ public sealed partial class App
         /// <returns>0 on success</returns>
         public static async Task<int> Main(string[] args)
         {
+            // display version info
+            // there is a bug in System.CommandLine that causes the built-in version handler to fail
+            if (args != null && args.Length > 0 && args.Contains("--version"))
```
Comment (jofultz) [inline src/app/Program.cs:42]: I think if the Length is 0 Contains will still return false.  So, might be able to take out the Length check.

File: src/app/WebValidation/Main.cs [hunk 4]
```diff
@@ -142,7 +142,20 @@ public async Task<int> RunOnce(Config config, CancellationToken token)
                     }
                 }
 
-                if (!config.JsonLog)
+                // write test summary in xml
+                if (config.XmlSummary)
+                {
+                    // todo - implement standard xml
```
Comment (jofultz) [inline src/app/WebValidation/Main.cs:148]: Does this code complete the todo?  If not, maybe pull out the todo into a work item.

File: src/app/WebValidation/Main.cs [hunk 5]
```diff
@@ -584,45 +600,57 @@ private void LogToConsole(Request request, ValidationResult valid, PerfLog perfL
...
 
-            if (config.JsonLog)
+            switch (config.LogFormat)
             {
-                Console.WriteLine(perfLog.ToJson(config.VerboseErrors));
-            }
+                case LogFormat.Json:
+                    Console.WriteLine(perfLog.ToJson(config.VerboseErrors));
+                    break;
+                case LogFormat.Tsv:
```
Comment (jofultz) [inline src/app/WebValidation/Main.cs:608]: Consider moving the code for the LogFormat.Tsv case into a separate private method for readability.

**Review by jofultz — APPROVED** [review]

File: src/app/WebValidation/Main.cs (hunk 4 above)
Comment (bartr) [reply src/app/WebValidation/Main.cs:148]: there is already a work item on the board

**Pull request merged**